nixos/modules/kairos.nix

```diff
@@ -0,0 +1,68 @@
+{ lib, pkgs, config, ... }:
+let
+  inherit (lib)
+    types
+    mkOption
+    mkIf
+    mkMerge
+    mkEnableOption
+    mdDoc
+    ;
+  cfg = config.services.kairos;
+in
+{
+  options.services.kairos = {
+
+    enable = mkEnableOption (mdDoc "kairos");
+
+    package = mkOption {
+      type = types.package;
+    };
+
+    port = mkOption {
+      type = types.port;
+      default = 60000;
+      example = 60000;
+      description = mdDoc ''
+        Port to listen on.
+      '';
+    };
+
+    logLevel = mkOption {
+      type = types.enum [
+        "error"
+        "warn"
+        "info"
+        "debug"
+        "trace"
+      ];
+      default = "info";
+      description = ''
+        The log-level that should be used.
+      '';
+    };
+  };
```

Comment:
This should have a string variant for supporting more complex log levels `RUST_LOG=[target][=][level][,...]`
https://docs.rs/env_logger/0.11.1/env_logger/#enabling-logging

Reply:
Since we don't need more sophisticated logging configuration yet, I would leave it as is to avoid spending the amount of work needed to replicate this is in nix code.